pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -7,6 +7,46 @@
             "#import": [ "base" ]
         },
 
+        "android": {
+            "#import": [ "any" ]
+        },
+        "android-arm64": {
+	           "#import": [ "any" ]
+        },
+        "android-mips": {
+            "#import": [ "any" ]
+        },
+
+        "android.21": {
+	           "#import": [ "android" ]
+	       },
+        "android.21-arm64": {
+	           "#import": [ "android.21", "android-arm64" ]
+        },
+        "android.21-mips": {
+	           "#import": [ "android.21", "android-mips" ]
+        },
+
+        "android.22": {
```

Comment:
I would keep API level 21 as the baseline - .NET Core will work on API level 21 so I would include it in the runtime.json file. You could add 22, 23, 24 and 25 as well.

Apart from remove MIPS, I would also add ARM. (so you'll have both arm and arm64).

I think the best way forward would be to update the PR, the discussion will be more tangible once I see the updated files.